Update instructions to enable Orchestrator for Agent v7.51+ (#22006)
DataDog/documentation: content/en/infrastructure/faq/set-up-orchestrator-explorer-daemonset.md

3. The Process Agent, which runs in the Agent DaemonSet, must be enabled and running (it doesn't need to run the process collection), and configured with the following options:

    ```yaml
    containers:
        - name: process-agent
          env:
          - name: DD_ORCHESTRATOR_EXPLORER_ENABLED
            value: "true"
    ```

>     For Agent versions 7.51.0+, the orchestrator check runs on the Agent container instead of the Process Agent container so configure the following on the Agent container instead:

Reviewer: ```suggestion
    For Agent versions 7.51.0+, the orchestrator check runs on the Agent container instead of the Process Agent container. To configure the Agent container:
```

    ```yaml
    containers:
        - name: agent
          env:
          - name: DD_ORCHESTRATOR_EXPLORER_ENABLED
            value: "true"
    ```

> 5. (Optional) Set collectors under instances section to specify the resources to be collected. Create `orchestrator.yaml` in ConfigMap. Sample configuration:

Reviewer: ```suggestion
4. (Optional) Set collectors under instances section to specify the resources to be collected. Create `orchestrator.yaml` in ConfigMap. Sample configuration:
```
